src/utils.js

```diff
@@ -96,6 +96,10 @@ function xhrRequest(url, callback, method){
       callback(error, error.message);
     }
   };
+  // triggered by network problems
+  xhr.onerror = function () {
+    callback({ status: 0, message: "", object: xhr }, "");
+  };
```

Comment:
Are you trying to silence the 404 etc. error messages from checking for DNT policies in the console? If so, bad news, this is deliberately impossible by design (at least in chrome).